web/blog/vast-v3.0/index.md

```diff
@@ -0,0 +1,349 @@
+---
+draft: true
+title: VAST v3.0
+description: VASTQL Refreshments
+authors: [dominiklohmann, mavam]
+date: 2023-01-20
+image: /path/to/image.png
+tags: [release, vastql, cef, performance, introspection, regex]
+---
+
+[VAST v3.0][github-vast-release] is out. The release brings some major updates
+to VAST's query language, including the ability to search using regular
+expressions.
+
+[github-vast-release]: https://github.com/tenzir/vast/releases/tag/v3.0.0
+
+<!--truncate-->
+
+## VASTQL Upgrades
+
+The 3.x series of VAST releases is all about the **VAST** **Q**uery
+**L**anguage. To kick this off, we've done some breaking changes to the language
+that we've wanted to do in quite a while. Here's a summary:
+
+1. Several built-in types have a new name:
+   - `int` → `int64`
+   - `count` → `uint64`
+   - `real` → `double`
+   - `addr` → `ip`
+   
+   The old names are still supported for the time being, but trigger a warning
+   on startup. We will remove support for the old names in a future release.
+
+2. The match operator `~` and its negated form `!~` no longer exist. Use queries
+   of the form `lhs == /rhs/` and `lhs != /rhs/` to perform searches with
+   regular expressions. Such queries now work for all string fields in addition
+   to the previously supported `#type` meta extractor.
+
+3. Queries of the form `#field == "some.field.name"` no longer exists to check
+   for the existence of a field. Use `some.field.name != nil` instead.
+
+4. The boolean literal values `T` and `F` are now `true` and `false`
+   respectively. For example the query `suricata.alert.alerted == T` is no
+   longer valid; use `suricata.alert.alerted == true` instead.
+
+Our goal is for these changes to make the query language feel more natural to
+our users. We've got [big plans][rfc-001] on how to extend it—and this felt very
+much necessary as a preparatory step to making the language more useful.
```

Comment:
Is it on purpose that you don't mention the term "pipeline" here?

Reply:
Yes. We've simply not yet been clear on the terminology, and it's somewhere between VASTQL the query language and VASTQL pipelines.

I think this is much more a question for Oliver in terms of branding, but I wanted to make him do a pass over this announcement anyways after we've got all the content we want and before we publish it.